test/unit/matrix/test_matrix.cpp

```diff
@@ -1077,6 +1078,8 @@ TYPED_TEST(MatrixTest, GPUCopy) {
   using GPUMemoryViewT = dlaf::memory::MemoryView<TypeParam, Device::GPU>;
   using GPUMatrixT = dlaf::Matrix<TypeParam, Device::GPU>;
 
+  hpx::cuda::experimental::enable_user_polling p;
+
```

Comment:
This is due for removal. I've been bitten by forgetting to have this too many times... I'm looking into what we can do in HPX to simplify using it.

Reply:
The need for enabling polling is not gone, but I've moved this to `dlaf::initialize`.